app/lib/lnd/methods/index.js

```diff
@@ -107,12 +107,17 @@ export default function(lightning, log, event, msg, data) {
       //   }
       // ]
       Promise.all(
-        [channelController.listChannels, channelController.pendingChannels].map(func => func(lnd))
+        [
+          channelController.listChannels,
+          channelController.pendingChannels,
+          channelController.closedChannels
+        ].map(func => func(lnd))
       )
         .then(channelsData =>
           event.sender.send('receiveChannels', {
             channels: channelsData[0].channels,
-            pendingChannels: channelsData[1]
+            pendingChannels: channelsData[1],
+            closedChannels: channelsData[2].channels
```

Comment:
is `channelsData[2]` always set?

Reply:
Yes - `channelsData[2]` is the spread from `Promise.all` above